templates/default/slurm_parallelcluster_gres.conf.erb

```diff
@@ -0,0 +1,2 @@
+# Sample dummy node gres entry
```

Comment:
shall we add the same comment set by sqswatcher or just say that this file will be generated by sqswatcher at runtime?